src/sdk/pynni/tests/test_compressor_torch.py

```diff
@@ -234,20 +234,24 @@ def test_torch_QAT_quantizer(self):
         model.relu = torch.nn.ReLU()
         quantizer = torch_compressor.QAT_Quantizer(model, config_list)
         quantizer.compress()
+
         # test quantize
         # range not including 0
         eps = 1e-7
         weight = torch.tensor([[1, 2], [3, 5]]).float()
-        quantize_weight = quantizer.quantize_weight(weight, model.conv2)
+        model.conv2.module.weight.data = weight
+        quantizer.quantize_weight(model.conv2)
         assert math.isclose(model.conv2.module.scale, 5 / 255, abs_tol=eps)
         assert model.conv2.module.zero_point == 0
         # range including 0
         weight = torch.tensor([[-1, 2], [3, 5]]).float()
-        quantize_weight = quantizer.quantize_weight(weight, model.conv2)
+        model.conv2.module.weight.data = weight
+        quantizer.quantize_weight(model.conv2)
         assert math.isclose(model.conv2.module.scale, 6 / 255, abs_tol=eps)
```

Comment:
The current quantizer test case is a little bit weak. It only checks:
1. no exception when run `quantizer.compress`
2. `quantize_weight` function

Can we add some `assert`s to check whether the weights and bias are quantized to expected range after `compress`?

Reply:
agree, @linbinskn